repositories.bzl

```diff
@@ -62,6 +62,10 @@ DIST_ARCHIVE_REPOS = [get_canonical_repo_name(repo) for repo in [
     "rules_foreign_cc+",
     "rules_fuzzing+",
     "rules_swift+",
+    "grpc-java++grpc_java_repositories_extension+com_github_cncf_xds",
```

Comment:
It looks like we are getting all dependencies from maven via rules_jvm_external, we do we need those?